clients/cloud/kotlin/src/main/kotlin/io/confluent/examples/clients/cloud/ProducerExample.kt

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:JvmName("ProducerExample")
+
+package io.confluent.examples.clients.cloud
+
+import io.confluent.examples.clients.cloud.model.DataRecord
+import io.confluent.examples.clients.cloud.util.loadConfig
+import io.confluent.kafka.serializers.KafkaJsonSerializer
+import org.apache.kafka.clients.admin.AdminClient
+import org.apache.kafka.clients.admin.NewTopic
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerConfig.*
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.clients.producer.RecordMetadata
+import org.apache.kafka.common.errors.TopicExistsException
+import org.apache.kafka.common.serialization.StringSerializer
+import java.util.*
+import java.util.concurrent.ExecutionException
+
+// Create topic in Confluent Cloud
+fun createTopic(topic: String,
+                partitions: Int,
+                replication: Short,
+                cloudConfig: Properties) {
+  val newTopic = NewTopic(topic, partitions, replication)
+
+
+  try {
+    with(AdminClient.create(cloudConfig)) {
+      createTopics(listOf(newTopic)).all().get()
+    }
+  } catch (e: Exception) {
```

Comment:
`ExecutionException`, following the java example, would be marginally better (acts as documentation if nothing else).

side note: the java example shouldn't really catch `InterruptedException` here, since the catch block does nothing with it.

Reply:
addressed